.github/ISSUE_TEMPLATE/feature-request--new-addition-to-addons-.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request (new addition to addons)
```

Comment:
Is it better to remove `new addition to addons`? Because people might just request to add / change behavior of existing function, rather than create a new function.

Reply:
yes feature-request.md could include both changes/additions.

Reply:
Thanks done